docs/USAGE.md

```diff
@@ -83,19 +89,19 @@ After installing the application you must authorize the application with your On
 
 You will be asked to open a specific URL by using your web browser where you will have to login into your Microsoft Account and give the application the permission to access your files. After giving permission to the application, you will be redirected to a blank page. Copy the URI of the blank page into the application.
 ```text
-[user@hostname ~]$ onedrive 
+[user@hostname ~]$ onedrive
```

Comment:
My editor also removed a bunch of extra whitespaces. It is better to review this PR with the `Hide whitespace changes` option turned on (or by adding `?w=1` to the PR url)